src/notifier.js

```diff
@@ -1,21 +1,16 @@
 import { notification } from './store.js'
 
-export function send (message, type = 'default', timeout) {
+export const send = (message, type = 'default', timeout) =>
```

Comment:
We're setting a default and not allowing the possibility of overriding it on a per-notification basis. I think we should add this as a parameter to these.

In fact since we now have multiple options this might be better as:

`msg, options` where options contains `timeout` and `closeable`

Reply:
That's something that i also felt was a bit off, but wasn't sure whether should be the default behavior or not

Taking into consideration that the options are going to be merged into a single parameter, should we keep all four mode functions (danger, warning, info and success), or may i refactor this file to a single `send` function, in a way the notification type will be passed to the options object:

```js
export const send = (message, options) =>
  notification.set({ message, options })
```
is called as:

```svelte
<button on:click={() => notifier.send('hello', { type: 'danger' })}>Notify</button>
```

or, we can keep all the four object modes, and just adapt their syntax to the `send` parameters changes, like the example below:

```js
export const danger = (msg, options) =>
  send(msg, { type: 'danger', ...options })

export const warning = (msg, options) =>
  send(msg, { type: 'warning', ...options })

export const info = (msg, options) =>
  send(msg, { type: 'info', ...options })

export const success = (msg, options) =>
  send(msg, { type: 'success', ...options })
```

by following this way, we keep the process of calling notifications basically unchanged, but, i personally think that by adhering to this, it will come with a bit of code duplication as a trade-off, but that's just me, choices like these are up to you :)

Reply:
I would suggest we keep the existing ones, since:

```js
notification.danger('foo')
```

is far more readable and easier to type than:

```js
notifications.send('foo', { type: 'danger' })
```